src/utils/stylelint/warning-to-diagnostic.ts

```diff
@@ -48,8 +48,7 @@ export function warningToDiagnostic(
 	const start = Position.create(warning.line - 1, warning.column - 1);
 	const end = Position.create(warning.line - 1, warning.column);
 
-	const ruleDocUrl =
-		rules?.[warning.rule] && `https://stylelint.io/user-guide/rules/${warning.rule}`;
+	const ruleDocUrl = rules?.[warning.rule]?.meta?.url;
```

Comment:
I'm not familiar with it, but does the warning object contain a url? If so, we can add a link to the rules documentation provided by the stylelint plugin by using the `warning.url`.

Reply:
Not `Warning`. `Rule` includes the `meta` property. See below:

- https://github.com/stylelint/stylelint/blob/e247454683f538e5b84fa8c80d0a8c402fdef494/types/stylelint/index.d.ts#L168-L173
- https://stylelint.io/developer-guide/plugins#the-anatomy-of-a-plugin

Reply:
If plugins could add `meta` for each rule, it would be so useful!

Reply:
I see 👍

Reply:
> If plugins could add meta for each rule, it would be so useful!

Yes. I think it would be very useful if we could link to it.